aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -155,6 +180,14 @@ module aptos_framework::transaction_validation {
             is_simulation,
             gas_payer_address
         )) {
+            assert!(
+                permissioned_signer::check_permission_consume(
+                    gas_payer,
```

Comment:
wondering if this is truly generic - if the gas payer is fee payer - can it use AA with delegated accounts and have permissioned signer here?

Reply:
When signer is passed here. permissioned signer will be passed.